roles/confluent.variables/defaults/main.yml

```diff
@@ -430,6 +448,9 @@ kafka_connect_ssl_enabled: "{{ssl_enabled}}"
 ### Boolean to enable mTLS Authentication on Connect
 kafka_connect_ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
 
+### Only use to customize directory Connect writes log files to. Default location is /var/log/kafka. NOTE- kafka_connect.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that Kafka Connect writes log files to. Default location is /var/log/kafka. NOTE- kafka_connect.appender_log_path is deprecated.